README.md

```diff
@@ -1,2 +1,72 @@
-# matama-s-portfolio
-iam matama, a software developer and this is my portfolio
+<a name="Daniel Matama"></a>
+
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Matama's Portfolio</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the project](#about-project)
+  - [🛠 built with](#built-with)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Usage](#usage)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [📝 License](#license)
```

Comment:
I noticed you have removed some sections from your readme file. Please make sure all missing sections in the readme template are present and they correctly navigate to their corresponding section 👍 👍 
![wggg](https://user-images.githubusercontent.com/73607512/205292222-2d8cfdc1-c217-4665-89b4-9a61f67c9376.PNG)